crates/telemetry/src/config.rs

```diff
@@ -0,0 +1,68 @@
+use envconfig::Envconfig;
+
+/// Provides configuration for the telemetry system.
+///
+/// Largely consists of OpenTelemetry variables defined
+/// [here](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration)
+/// but also has some Spin specific configuration.
+#[derive(Debug)]
+pub struct Config {
+    /// Disables OpenTelemetry for all signals (traces and metrics).
+    pub otel_sdk_disabled: bool,
+
+    /// Determines the verbosity of the OpenTelemetry tracing layer.
+    ///
+    /// This is a Spin specific value. This is what allows us to have different levels of verbosity
```

Comment:
If it's Spin-specific, do we document the recognised values?  Should we reference that documentation here?  Could it be an enum?

Reply:
Good call. I changed this to just directly use the underlying `tracing::Level` type that we parse it into.